en/docs/try-out/payment-initiation-service-flow.md

```diff
@@ -0,0 +1,601 @@
+    -H 'Signature: keyId="SN=de730ec5733ead1b,CA=1.2.840.113549.1.9.1=#16116d616c7368616e694077736f322e636f6d,CN=OpenBanking Pre-Production Issuing CA,OU=OpenBanking,O=WSO2 (UK) LIMITED,L=COL,ST=WP,C=LK",algorithm="rsa-sha256", headers="digest x-request-id date psu-id",signature=vygh2dDrkoku18LcDgIBA4iKU9V/P/LZCb0OZC3d+yoIygc/ceehj3K13MbTR4m3F+QOC4ksRCuBRxwG3vtp4WYBf3rFckTBxOAlH0yhfbZqg0f5smj/H3CzhuxsxtIIo2ny+NOYbs3ycWfNseB6lL3Pc2x/ZZkEqBT3kTq4zrrRedFkIRZmow8nhCJ00YvWqqgV6MusFl1wI3JRJ4R1nfBEKs2aQTrao3XM29mwulXnOZMg78KSi55W+qbKjqBY9N33ksEGlt2YNAMI8Ap+hh4kyVY7YYwmkSKlXtqz4dZU4f3UMsHz4pk0iEnXM/KCR0PGa8pubYV3lt4wDE/ZXA==' \
+    -H 'TPP-Signature-Certificate: <SIGNATURE_CERTIFICATE>' \
+    -H 'Content-Type: application/json; charset=UTF-8' \
+    -H 'Authorization: <APPLICATION_ACCESS_TOKEN>' \
+    --cert <PUBLIC_KEY_FILE_PATH> --key <PRIVATE_KEY_FILE_PATH> \
+    -d '{
+       "batchBookingPreferred": true,
+       "debtorAccount": {
+           "iban": "DE12345678901234567890",
+           "currency": "EUR"
+       },
+       "requestedExecutionDate": "2022-03-16",
+       "payments": [
+           {
+               "instructedAmount": {
+                   "currency": "EUR",
+                   "amount": "123.50"
+               },
+               "creditorName": "Merchant123",
+               "creditorAccount": {
+                   "iban": "DE98765432109876543210"
+               },
+               "remittanceInformationUnstructured": "Ref Number Merchant"
+           },
+           {
+               "instructedAmount": {
+                   "currency": "EUR",
+                   "amount": "123.50"
+               },
+               "creditorName": "Merchant",
+               "creditorAccount": {
+                   "iban": "DE23100120020123456789"
+               },
+               "remittanceInformationUnstructured": "Ref Number Merchant"
+           }
+       ]
+    }'
+    ```
+   
+    ``` tab="Response"
+    {
+       "transactionStatus":"RCVD",
+       "chosenScaMethod":{
+          "authenticationVersion":"1.0",
+          "name":"SMS OTP on Mobile",
+          "authenticationType":"SMS_OTP",
+          "explanation":"SMS based one time password",
+          "authenticationMethodId":"sms-otp"
+       },
+       "_links":{
+          "scaStatus":{
+             "href":"/v1/bulk-payments/sepa-credit-transfers/9e6229ac-a888-44ac-80cc-d384d8e20e73/authorisations/eb75d62e-5722-4233-aa66-860aaa818366"
+          },
+          "scaOAuth":{
+             "href":"https://localhost:8243/.well-known/openid-configuration"
+          },
+          "self":{
+             "href":"/v1/bulk-payments/sepa-credit-transfers/9e6229ac-a888-44ac-80cc-d384d8e20e73"
+          },
+          "status":{
+             "href":"/v1/bulk-payments/sepa-credit-transfers/9e6229ac-a888-44ac-80cc-d384d8e20e73/status"
+          }
+       },
+       "paymentId":"9e6229ac-a888-44ac-80cc-d384d8e20e73"
+    }
+    ```
+   
+    #### Multi-Currency Payments 
```

Comment:
A new configuration will be introduced to payments multi-currency flow to enable or disable the currency validation when doing a payment. This will be implemented and updated. Since it changes the behavior of the payment, we can hide this section for now in the docs.

Reply:
Fixed in 7bfa39b2fba54a9ecb39b651ec24854dea85354f